packages/stream-tile/src/tile-document.ts

```diff
@@ -344,8 +344,13 @@ export class TileDocument<T = Record<string, any>> extends Stream {
       newContent = this.content
     }
 
-    if (header.controllers && header.controllers?.length !== 1) {
-      throw new Error('Exactly one controller must be specified')
+    if (header.controllers) {
+      if (header.controllers?.length !== 1) {
+        throw new Error('Exactly one controller must be specified')
+      }
+      if (header.controllers[0] === null || header.controllers[0] === undefined || header.controllers[0] == '') {
```

Comment:
```suggestion
      if (!Boolean(header.controllers[0])) {
```
[Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean) should suffice for this.